source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/Fixtures/ChargesFunctionAppFixture.cs

```diff
@@ -103,13 +98,7 @@ protected override async Task OnInitializeFunctionAppDependenciesAsync(IConfigur
             Environment.SetEnvironmentVariable(EnvironmentSettingNames.DataHubListenerConnectionString, ServiceBusResourceProvider.ConnectionString);
             Environment.SetEnvironmentVariable(EnvironmentSettingNames.DataHubManagerConnectionString, ServiceBusResourceProvider.ConnectionString);
 
-            var postOfficeTopic = await ServiceBusResourceProvider
-                .BuildTopic(ChargesServiceBusResourceNames.PostOfficeTopicKey).SetEnvironmentVariableToTopicName(EnvironmentSettingNames.PostOfficeTopicName)
-                .AddSubscription(ChargesServiceBusResourceNames.PostOfficeSubscriptionName)
-                .CreateAsync();
-
-            PostOfficeListener = new ServiceBusListenerMock(ServiceBusResourceProvider.ConnectionString, TestLogger);
-            await PostOfficeListener.AddTopicSubscriptionListenerAsync(postOfficeTopic.Name, ChargesServiceBusResourceNames.PostOfficeSubscriptionName);
+            await InitializePostOfficeAsync();
```

Comment:
I suggest
```suggestion
            await InitializeMessageHubAsync();
```
and renaming of PostOffice elsewhere too.

Reply:
At daily @bemwen said/accepted he would do it as it fits very well with his story.
It's on the contrary not the focus of this story - and would also cause some issues regarding our obsolete mocking of PostOffice/MessageHub.

Reply:
Btw, this is explicitly about the obsolete stuff. I wouldn't like it to be named MessageHub as it would make it slightly more difficult to clean up.